content/docs/buildpacks/language-family-buildpacks/java.md

```diff
@@ -146,14 +155,17 @@ pack build samples/java \
 The resulting application image will be identical to that built in the Building with Maven example.
 
 ## About the JVM
+
 The Java Buildpack uses the [BellSoft Liberica][liberica] implementations of the JRE and JDK. JVM installation is handled by the [BellSoft Liberica Buildpack][bp/bellsoft-liberica]. The JDK will be installed in the build container but only the JRE will be contributed to the application image.
 
-See the [homepage][[bp/bellsoft-liberica]] for the Bellsoft Liberica Buildpackfor a full set of configuration options.
+See the [homepage][bp/bellsoft-liberica] for the Bellsoft Liberica Buildpackfor a full set of configuration options.
```

Comment:
```suggestion
See the [homepage][bp/bellsoft-liberica] for the Bellsoft Liberica Buildpack for a full set of configuration options.
```
this was my typo from before but we might as well fix it while we are looking at it :)